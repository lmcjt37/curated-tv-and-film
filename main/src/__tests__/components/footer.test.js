import React from 'react';
import ReactDOM from 'react-dom';
import Footer from '../../components/footer';
import render  from 'react-test-renderer';

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<Footer />, div);
  ReactDOM.unmountComponentAtNode(div);
});

it('snapshot of initial component', () => {
    const component = render.create(<Footer />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});
