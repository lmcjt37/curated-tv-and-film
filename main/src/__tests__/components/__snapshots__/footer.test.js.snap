// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot of initial component 1`] = `
<footer
  id="footer"
>
  <div
    className="container"
  >
    <ul
      className="links"
    >
      <li>
        <a
          href="https://github.com/lmcjt37/curated-tv-and-film"
          rel="noopener noreferrer"
          target="_blank"
        >
          <span>
            Repository
          </span>
        </a>
      </li>
      <li>
        |
      </li>
      <li>
        <a
          href="https://github.com/lmcjt37/curated-tv-and-film/graphs/contributors"
          rel="noopener noreferrer"
          target="_blank"
        >
          <span>
            Contributors
          </span>
        </a>
      </li>
    </ul>
    <p
      className="version"
    >
      <a
        rel="noopener noreferrer"
        target="_blank"
      >
        build 
        debug
      </a>
    </p>
  </div>
</footer>
`;
